### PR TITLE
Blog on FOSDEM 2017 CfP

### PR DESCRIPTION
**Pull request by bproffitt**
Changes proposed in this pull request:

- Created CfP blog entry for FOSDEM 2017 devroom

- Blog is ready, but critical question: since we only have one day, do we want to do mentored programs this year?
I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): @bproffitt

This pull request needs review by: @doron-fediuck @lars_kurth

### CODE REVIEW

File: source/blog/2016-10-12-call-for-proposal-fosdem-2017.html.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+This devroom is a collaborative effort, and is organized by dedicated folks from projects such as OpenStack, Xen Project, Mesos, oVirt, and Foreman. We would like to invite all those who are involved in these fields to submit your proposals by November 18th, 2016.
+
+READMORE
+
+## About the Devroom
+
+The Virtualization & IaaS devroom will feature session topics such as open source hypervisors and virtual machine managers such as KVM, Xen Project, bhyve, and VirtualBox, and Infrastructure-as-a-Service projects such as Apache CloudStack, OpenStack, oVirt, OpenNebula, and Ganeti.
+
+This devroom will host presentations and labs that focus on topics of shared interest, such as KVM, libvirt, shared storage, virtualized networking, clustering and high availability, interfacing with multiple hypervisors, and scaling across hundreds or thousands of servers.
```
Comment (doron-fediuck) [inline source/blog/2016-10-12-call-for-proposal-fosdem-2017.html.md]: Labs???

Comment (bproffitt) [inline source/blog/2016-10-12-call-for-proposal-fosdem-2017.html.md]: Yeah, that was a two-day thing we thought about. Dropping.

Comment (bproffitt) [inline source/blog/2016-10-12-call-for-proposal-fosdem-2017.html.md]: +1

**Comment by bproffitt** [comment]
Merged #522

**Pull request merged**